test/js/database/PiskelDatabaseTest.js

```diff
@@ -44,13 +46,17 @@ describe('PiskelDatabase test', function () {
     var dbName = pskl.database.PiskelDatabase.DB_NAME;
     var req = window.indexedDB.deleteDatabase(dbName);
     req.onsuccess = done;
+
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
   });
 
   afterEach(function () {
     // Close the database if it was still open.
     if (piskelDatabase && piskelDatabase.db) {
       piskelDatabase.db.close();
     }
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
```

Comment:
Why was it necessary to extend the test timeout here, and how did you figure that out?

Reply:
I was having a lot of trouble with timeouts in this test on both this PR and #35. Adding this seemed to make it timeout significantly less but I didn't do any analysis to figure out what the appropriate timeout here would be.

Reply:
I played with this some more and the unit tests consistently fail without this extended timeout and consistently pass with it. This was happening on both my recent PRs so I don't think this is something broken by this change.